fedimint-derive/src/lib.rs

```diff
@@ -248,7 +248,15 @@ fn derive_enum_decode(ident: &Ident, variants: &Punctuated<Variant, Comma>) -> T
             // FIXME: make sure we read all bytes
             quote! {
                 #variant_idx => {
-                    let bytes: Vec<u8> = ::fedimint_core::encoding::Decodable::consensus_decode_from_finite_reader(d, modules)?;
+                    // FIXME: feels like there's a way more elegant way to do this with limited readers
+                    let bytes_res = ::fedimint_core::encoding::Decodable::consensus_decode_from_finite_reader(d, modules);
+                    let bytes: Vec<u8> = ::anyhow::Context::context(
```

Comment:
I don't understand why this isn't just `.context(....)` on the same line.

Reply:
Is putting `use anyhow::Context as _;` right here an issue?

Reply:
You are right, I could put that into the top-level derive fn that defines the decode fn. I got a bit hung up on not being able to `use` it where I need it since that would import it multiple times leading to errors.

Reply:
fixed